keyboards/westm/westm9/keymaps/default/keymap.c

```diff
@@ -0,0 +1,133 @@
+/* Copyright 2021 WestM
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include QMK_KEYBOARD_H
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+    [0] = LAYOUT_macropad( /* Base */
+        RGB_TOG, RGB_RMOD, RGB_MOD,
+        RGB_VAI, RGB_VAD, KC_6,
+        RGB_HUI, KC_1, KC_2
+    ),
+    [1] = LAYOUT_macropad( /* FN */
+        KC_1, KC_2, KC_3,
+        KC_4, KC_5, KC_6,
+        KC_7, KC_8, KC_9
+    )
+};
+
+#ifdef OLED_ENABLE
```

Comment:
This block should be moved the keyboard's c file.

Reply:
Oh, even if future revisions will be using the same one display? I just thought if I left it here then all versions would draw from the same place.

Reply:
Sorry, let me add the reasoning behind this. 

The reason for having it placed in the keyboard's c file, is that so it has a default behavior, and one doesn't need to be explicitly defined.  This is useful for users that want to customize their display... or not bother with it.  

Additionally, this allows firmware compiled by the QMK Configurator to have a default display rendered, too.   

Eg, it provides a better UX for users, overall.

Reply:
Ah okay, that makes sense!